src/session-client.ts

```diff
@@ -3,7 +3,17 @@ import { HashWordArray } from "./utils";
 
 export class SRPClientSession {
   constructor(private readonly routines: SRPRoutines) {}
-  public step1(userId: string, userPassword: string): SRPClientSessionStep1 {
+
```

Comment:
Nit: also add comment that current variable names greatly simplify matching of the code with RFC.